modules/images-configuration-registry-mirror.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration.adoc
+
+[id="images-configuration-registry-mirror_{context}"]
+= Configuring image registry repository mirroring
+Setting up container registry repository mirroring lets you:
+
+* Configure your {product-title} cluster to redirect requests
+to pull images from a repository on a source image registry and have
+it resolved by a repository on a mirrored image registry.
+* Identify multiple mirrored repositories for each target
+repository, to make sure that if one mirror is down, another
+can be used.
+
+Here are some of the attributes of repository mirroring in {product-title}:
+
+* Image pulls are resilient to registry downtimes
+* Disconnected clusters can request to pull
+images from critical locations (such as quay.io)
```

Comment:
We're not calling them "disconnected." They're "clusters in restricted networks" now.
Quay.io
> Avoid using parentheses in running text because text in parentheses is decreased in importance.

Reply:
Okay. Fixed. I don't mind the parentheses thing.